File: sprint2/By languages/Java/lesson7code1.java
import java.util.*;

class Stack {
  private List<String> items;

  public Stack() {
    items = new ArrayList<String>();
  }

  public void push(String item) {
    items.add(item);
  }

  public String pop() {
    return items.remove(items.size() - 1);
  }

  public String peek() {
    return items.get(items.size() - 1);
  }

  public int size() {
    return items.size();
  }
}

Stack stack = new Stack();
stack.push("apple");
stack.push("banana");
stack.push("orange");
stack.pop();

File: sprint2/By languages/kotlin/lesson7code1.kt
class Stack {
    private var items = mutableListOf<Any>()
    fun push(item: Any) {
        items.add(item)
    }
    fun pop(): Any? {
        return if (items.isNotEmpty()) items.removeAt(items.size - 1) else null
    }
    fun peek(): Any? {
        return if (items.isNotEmpty()) items.last() else null
    }
    fun size(): Int {
        return items.size
    }
}

fun main(args: Array<String>) {
    val stack = Stack()
    stack.push("apple")
    stack.push("banana")
    stack.push("orange")
    stack.pop()
}